File: ezSCORES/ezSCORES.Model/Referees.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ezSCORES.Model
{
	public class Referees
	{
		public int Id { get; set; }

		public string FirstName { get; set; } = null!;

		public string LastName { get; set; } = null!;

		public byte[]? Picture { get; set; }
	}
}

File: ezSCORES/UI/ezscores_desktop/lib/models/referees.dart
import 'package:json_annotation/json_annotation.dart';

part 'referees.g.dart';

@JsonSerializable()
class Referees {
	int? id;
	String? firstName;
	String? lastName;
	String? picture;

	Referees();

	factory Referees.fromJson(Map<String, dynamic> json) => _$RefereesFromJson(json);

	Map<String, dynamic> toJson() => _$RefereesToJson(this);
}